src/layouts/components/Editor/extensions/Iframe/IframeView.tsx

```diff
@@ -1,33 +1,37 @@
 import { Box, Text } from "@chakra-ui/react"
 import { NodeViewProps, NodeViewWrapper } from "@tiptap/react"
 
-export const IframeView = ({ node }: NodeViewProps) => {
+import { BlockWrapper } from "../../components/BlockWrapper"
+
+export const IframeView = ({ node, selected }: NodeViewProps) => {
   return (
-    <Box
-      as={NodeViewWrapper}
-      bg="#FAF594"
-      border="3px solid #0d0d0d"
-      borderRadius="0.5rem"
-      margin="1rem 0"
-      position="relative"
-      data-drag-handle
-    >
-      <Text
-        ml="1rem"
-        bgColor="#0d0d0d"
-        textStyle="caption-1"
-        fontWeight="bold"
-        color="#fff"
-        position="absolute"
-        top="0"
-        padding="0.25rem 0.75rem"
-        borderRadius="0 0 0.5rem 0.5rem"
+    <BlockWrapper name="Iframe" isSelected={selected}>
```

Comment:
nit: Might be good to standardise as "Embed" and also add this BlockWrapper to InstagramView and FormSGView

Reply:
oh didn't realise there were different views! will do, thanks for pointing it out